package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wmt-probation-rules",
-  "version": "0.1.12",
+  "version": "0.1.13",
```

Comment:
We're going to need to coordinate our versioning here.

Reply:
Yeh, I just bumped from `master`, but I'm guessing your PR will be in first, so I'll merge and rebump